src/base/bittorrent/torrentinfo.h

```diff
@@ -34,12 +34,15 @@
 #include <libtorrent/torrent_info.hpp>
 #include <libtorrent/version.hpp>
 
+#include "indexrange.h"
+
 class QString;
 class QUrl;
 class QDateTime;
 class QStringList;
 class QByteArray;
 template<typename T> class QList;
+template<typename T> class QVector;
```

Comment:
It works, of course. Decided that the file is so tiny that such optimisation is not significant, while, naturally, this ways it is more easy to use.